grakn-test-profiles/pom.xml

```diff
@@ -122,46 +120,5 @@
                 </dependency>
             </dependencies>
         </profile>
-
-        <profile>
-            <id>scaling</id>
```

Comment:
Why don't we need the scaling profile anymore? I am not even sure what it's used for.